### ISSUE TITLE
Got low accuracy results for Scannet per-point prediction

### ISSUE THREAD

**Issue by sylyt62** [microsoft/O-CNN#154]
I followed ScanNet Segmentation instructions, but got bad prediction results. 

Firstly, I downloaded scene0000_00 from training set and scene0707_00 from test set, ran `python tools/scannet.py --run process_scannet --path_in <scannet_folder>` to prepare the data, and ceated my own filelist.txt

Secondly,  changed config file (based on `seg_scannet_eval.yaml`, only changed in and out dir) and ran `python segmentation.py --config configs/seg_custom_eval.yaml`, using weights 00600.model.pth

Finally, converted the predictions to segmentation labels.

No errors jumped out, and got results in logs folder. But after I attach the predicted labels to ply file, seems like it can only distinguish walls and floor.

Did I miss something?

pic1: scene0000_00 predictions
![000_pred](https://user-images.githubusercontent.com/80388134/162926821-d7124a1a-df2f-4232-80f7-dc77352de3e4.png)

pic2: scene0000_00 labels
![000_label](https://user-images.githubusercontent.com/80388134/162926804-0f0640e2-7062-4f20-b217-30ab3f1c0a24.png)

scene0707_00 predictions
![070_pred](https://user-images.githubusercontent.com/80388134/162926835-25245223-f6fb-4b39-a767-bbf10157c0b0.png)

**Comment by wang-ps** [comment]
Could you forward the network on the validation dataset and get a mIoU around 74.0?
